controller/env/broker.go

```diff
@@ -174,7 +182,10 @@ func (broker *Broker) apiSessionCertificateHandler(args ...interface{}) {
 	})
 
 	if err != nil {
-		pfxlog.Logger().WithError(err).Error("could not process API Session certificate event, failed to query for parent API Session")
+		// If it's not found, it's because it was deleted, which
```

Comment:
"which ..." ? Not critical but it leaves me on the edge of my seat.

Reply:
Tune in for the thrilling conclusion in next week's commit :smile: 

Fixed